### PR TITLE
Create Workflow.ylm

### LINKED ISSUE #21: Contracts blockchain

### PR DESCRIPTION
**Pull request by Herand19**

### CODE REVIEW

**Review by Herand19 — COMMENTED** [review]
.

**Review by Herand19 — COMMENTED** [review]
.

**Review by Loky1s — APPROVED** [review]

**Review by Dioliode — APPROVED** [review]
DioX

**Review by Herand19 — COMMENTED** [review]
.

File: .github/workflows/Workflow.ylm [hunk 1]
```diff
@@ -0,0 +1 @@
+#Creating Pages deployment with payload: { "artifact_id": 1383468329, "pages_build_version": f63ecbdae79de395c3000f126bde06afb83e253d
```
[the remark this answers is not in the document]
Comment (Loky1s) [reply .github/workflows/Workflow.ylm:1]: Why is all my money going to starlight? What are you doing my money?
 You gonna rob me too like the rest of everybody? I thought you're trying to help me. You aint trying to help me. I'm trying to help yourself to my money. Why is all my money going to starlight enterprise

**Comment by Loky1s** [comment]
Why is all my money going to starlight enterprise?Are you taking my money
too cause?I don't know how to get it out, so you're gonna take it.You're
not gonna help me.You're gonna take you off and you're go screw me
around.I'm not letting it happen, I'm not letting it happen.I'm just not
letting it happen

ds

**Comment by Loky1s** [comment]
Why are you taking my money?November 20 20th whenever I started.That's when
I bought the man if I ever find you if I ever lay my eyes on you.You are
fucking dead

ds

**Comment by Loky1s** [comment]
I didn't approve of anything.I'll stole much of my money.You'll trick me.I knew you had to 560 million dollars out of the same day as soon as you'll trick me.Start your little sticking program

File: .github/workflows/Workflow.ylm (hunk 1 above)
[the remark this answers is not in the document]
Comment (Loky1s) [reply .github/workflows/Workflow.ylm:1]: Just just a scumbag, that's all I can say you're a f****** scumbag.

**Review by Herand19 — COMMENTED** [review]
.

**Review by Herand19 — COMMENTED** [review]
Ok

**Pull request merged**